webclient/discovery/src/test/java/io/helidon/webclient/discovery/TestUriEncoding.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.webclient.discovery;
+
+import java.net.URI;
+
+import io.helidon.common.uri.UriEncoding;
+import io.helidon.webclient.api.ClientUri;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+class TestUriEncoding {
+
+    private TestUriEncoding() {
+        super();
+    }
+
+    @Test
+    void testDecode() {
+        ClientUri c = ClientUri.create();
```

Comment:
This test does not seem to belong here at all - this should be tested as part of `io.helidon.webclient.api` module.
If this test is missing, please add it. If it exists, please remove this one.

Reply:
Resolved in https://github.com/helidon-io/helidon/pull/10786/commits/94c1613005ba25167e31dd262561ef2eedaa63b3.